tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -15,6 +15,25 @@
 
 
 class ImportCluster(flows.BaseFlow):
+    def _probe_and_mark_provisioner(self, node_list, integration_id):
+        for node in node_list:
```

Comment:
Instead of iterating over list of nodes and loading every node's NodeContext

1) fetch the list of node for tag "provisioner/gluster"  from `/indexes/tags/provisioner/gluster`

The from set(node_list) - set(list_of_node_from_step_1) , you can figure out the node with "provisioner/gluster" tag